streaming-binlog/src/main/java/org/openmrs/analytics/FhirConverter.java

```diff
@@ -92,8 +101,10 @@ public void process(Exchange exchange) {
 			return;
 		}
 		if (payload.get("uuid") == null) {
-			log.error(String.format("No uuid column for table %s ignoring payload %s ", table, payload));
-			return;
+			String resultUuid = getUuid(config.getParentTable(), config.getParentForeignKey(), config.getChildPrimaryKey(),
+			    payload);
+			
+			payload.put("uuid", resultUuid);
```

Comment:
I think it is better not to change the `payload` and instead set the `uuid` directly here (you can move line 109 before this `if` block.

Reply:
@gitcliff did you see @bashir2 comment ??